manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: read-openshift-config-managed
```

Comment:
Probably `name: console-operator` to match the rest.

Reply:
` oc get rolebinding -n openshift-console` should probably list `console-operator` as a rolebinding.

Reply:
example:
```
oc get rolebinding -n openshift-console
NAME                    AGE
console-operator        35h
system:deployers        35h
system:image-builders   35h
system:image-pullers    35h
```